Gemfile.lock

```diff
@@ -0,0 +1,135 @@
+PATH
```

Comment:
We still gotta delete this file since we shouldn't check in the lock file for gems.